src/scene/shader-lib/glsl/chunks/gsplat/frag/gsplat-copy-to-workbuffer.js

```diff
@@ -0,0 +1,92 @@
+// fragment shader to copy splats in any supported format to MRT work-buffer
```

Comment:
We have that ugly camel case naming for chunks. Shouldn't we adhere to that?

Reply:
good point, renamed those chunks, thanks